README.md

```diff
@@ -426,11 +426,11 @@ The **`generatePipeline | gpl`** task will generate CI pipeline inside the folde
    
  ## Generate Acceptance Test
  
-The **`generateAcceptanceTest | gat`** task will generate subproject by [karate framework](https://github.com/intuit/karate)  inside the folder "./deployment/",this task does not have  required parameters.
+The **`generateAcceptanceTest | gat`** task will generate subproject by [karate framework](https://github.com/intuit/karate)  inside the folder "./deployment/", this task has one required parameter `name`.
    
 ```shell
-      gradle generateAcceptanceTest
-      gradle gat 
+      gradle generateAcceptanceTest --name=[acceptanceTestProjectName]
+      gradle gat --name=[acceptanceTestProjectName] 
```

Comment:
why required? couldn't you have a default value and make it an optional parameter?

Reply:
Sure, I set as a default name for the acceptance test project the value acceptanceTest, now the name parameter is optional